src/ReinforcementLearningExperiments/test/runtests.jl

```diff
@@ -15,6 +23,8 @@ run(E`JuliaRL_VPG_CartPole`)
 run(E`JuliaRL_MPODiscrete_CartPole`)
 run(E`JuliaRL_MPOContinuous_CartPole`)
 run(E`JuliaRL_MPOCovariance_CartPole`)
+run(E`JuliaRL_IDQN_TicTacToe`)
+@require PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0" run(E`JuliaRL_DQN_MPESimple`)
```

Comment:
this one is cause of failure. We want to run it if PyCall is present and pettingzoo is installed in python env. Will asap push a suggestion for this.